ovos_padatious/simple_intent.py

```
@@ -12,81 +12,108 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import List, Tuple, Optional
```

Ruff

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)